common/subset-tests.js

```diff
@@ -0,0 +1,27 @@
+// Only test a subset of tests with, e.g., ?1-10 in the URL
+// Sample usage:
+// let currentSubTest = 0;
```

Comment:
Hmmm... so every test is going to end up with this sample usage. What do you think about altering (or adding) the API to take care of this, i.e. just exposing a function (e.g. "subsetTest") that takes a closure, which handles the counter/test for the author?
```

Reply:
I think this is more flexible and more straightforward to apply to existing tests, since it can vary a lot how tests are being set up. But maybe we can support both somehow...